__tests__/unit-tests/routes/activity/activity.controller.test.ts

```diff
@@ -0,0 +1,151 @@
+import request from 'supertest';
+import express, { Application } from 'express';
+import mongoose from 'mongoose';
+import ActivityController from '@routes/activity/activity.controller';
+import { ActivityLog, User } from '@models';
+import { DatabaseHelpers } from '../../../helpers/database-helpers';
+
+// Set the timeout of the test to 60 seconds
+jest.setTimeout(60000);
+
+/** Express application instance */
+const app: Application = express();
```

Comment:
@TaiKamilla 
you should check how tests were done in Ouirace
unit testing doesn't require setting up the whole application server for a single test, it's about testing specific units of the code

what you wrote is more integration tests that unit tests